services/minespace-web/src/components/dashboard/mine/variances/VariancesTable.js

```diff
@@ -54,6 +56,11 @@ export class VarianceTable extends Component {
     }
   };
 
+  // NOTE: Luke/Brian: The expired "all red" styling is to be REMOVED.
```

Comment:
is this a back and forth conversation? can this either be removed or summarized into a single direction for the next developer?

Reply:
It's meant as a note, from Luke and Brian, for the developer.

Reply:
IMO I think it's fine for this PR for this sprint, but these should all be removed by the end of this sprint.

Reply:
That was my intention :)